wagtail/wagtailadmin/rich_text.py

```diff
@@ -31,6 +32,15 @@ def value_from_datadict(self, data, files, name):
             return None
         return DbWhitelister.clean(original_value)
 
+    @property
+    def media(self):
+        return Media(js=[
+            static('wagtailadmin/js/vendor/hallo.js'),
+            static('wagtailadmin/js/hallo-plugins/hallo-wagtaillink.js'),
+            static('wagtailadmin/js/hallo-plugins/hallo-hr.js'),
+            static('wagtailadmin/js/hallo-plugins/hallo-requireparagraphs.js'),
+        ])
+
```

Comment:
It is slightly different on Django 1.8 though, where it appends ``STATIC_URL`` to the beginning: https://github.com/django/django/blob/stable/1.8.x/django/forms/widgets.py#L73